### PR TITLE
ENH: pandas Series and Index fallbacks

### LINKED ISSUE #92: ENH: pandas series and index fallbacks

### PR DESCRIPTION
**Pull request by UranusSeven**
## Related issue number

Resolves #92 

## Check code requirements

- [x] tests added / passed (if needed)
- [x] Ensure all linting tests pass

### CODE REVIEW

File: python/xorbits/pandas/pandas_adapters/core.py [hunk 1]
```diff
@@ -81,51 +81,72 @@ def _get_output_type(func: Callable) -> MarsOutputType:
...
-            )
+            if hasattr(one_chunk_entity, "map_chunk"):
+                return _map_chunk(one_chunk_entity, skip_infer=False)
+            else:
+                return _spawn(one_chunk_entity)
         except TypeError:
+            import traceback
+
+            traceback.print_exc()
+
```
Comment (aresnow1) [inline python/xorbits/pandas/pandas_adapters/core.py:137]: What's these for?

File: python/xorbits/pandas/pandas_adapters/core.py [hunk 2]
```diff
@@ -81,51 +81,72 @@ def _get_output_type(func: Callable) -> MarsOutputType:
...
         try:
-            ret = one_chunk_data.map_chunk(
-                lambda x, *args, **kwargs: getattr(x, func_name)(*args, **kwargs),
-                args=args,
-                kwargs=kwargs,
-            )
+            if hasattr(one_chunk_entity, "map_chunk"):
+                return _map_chunk(one_chunk_entity, skip_infer=False)
+            else:
+                return _spawn(one_chunk_entity)
```
Comment (aresnow1) [inline python/xorbits/pandas/pandas_adapters/core.py:132]: What happens if `_spawn` raises `TypeError`?

File: python/xorbits/pandas/pandas_adapters/core.py (hunk 1 above)
Comment (UranusSeven) [reply python/xorbits/pandas/pandas_adapters/core.py]: Removed.

File: python/xorbits/pandas/pandas_adapters/core.py (hunk 2 above)
Comment (UranusSeven) [reply python/xorbits/pandas/pandas_adapters/core.py]: Fixed.

**Review by qianduoduo0904 — APPROVED** [review]
LGTM

**Review by aresnow1 — APPROVED** [review]
LGTM

**Pull request merged**